draft-ietf-httpbis-sfbis.md

```diff
@@ -191,6 +191,13 @@ being used.</t>
 </artwork>
 </blockquote>
 
+Note that because the definition of a Strutured Field references a specific RFC for Structured Fields, the types available for use in its value are limited to those defined in that RFC. For example, a field whose definition references this document can have a value that uses the Date type ({{date}}), whereas a field whose definition references RFC 8941 cannot, because it will be treated as invalid (and therefore discarded) by implementations of that specification.
+
+Also note that this limitation also applies to future extensions to a field; for example, a field that is defined with reference to RFC 8941 cannot use the Date type, because some recipients might still be using an RFC 8941 parser to process it.
+
+However, this document is designed to be backwards-compatible with RFC 8941; a parser that implements the requirements here can also parse valid Structured Fields whose definitions reference RFC 8941.
+
```

Comment:
Hm, that's very wordy, and this section is already quite wordy.

Reply:
I know. I don’t like it much for that reason. But maybe this is worth the text. Slapping a section header on might help. “Backwards Compatibility” might be a good title.

Reply:
I cannot in good confidence say that any of this adds clarity, but I will not go as far as to call it surpluage, because there is a point to be made here.

I think what bugs me, is that having a SFbis parser return an unexpected and illegal sf-date is not materially different from having a 8941 parser return sf-decimal where sf-integer was expected.

In the end, the responsibility for validating the header can only fall on the application code.

Maybe we should simply write that, and use the 8941-defined-SFbis-parser-returning-sf-date example to illustrate a detail people might otherwise miss ?

Reply:
See proposal below.